source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -50,9 +51,16 @@ def get(self, request, instance, shape=None):
         if not self.external_api:
             raise TechnicalError('global autocomplete not configured')
 
-        url = '{endpoint}?q={q}&limit={count}'.format(endpoint=self.external_api,
-                                                      q=request['q'],
-                                                      count=request['count'])
+        params = {
+            "q": request["q"],
+            "count": request["count"]
+        }
+
+        if request["from"]:
+            params["lon"], params["lat"] = self.get_coords(request["from"])
+
+        url = '{endpoint}?{query}'.format(endpoint=self.external_api, query=urlencode(params))
```

Comment:
I think it's better to use the `requests` build in params no ?

`requests.post(url, timeout=self.timeout, json=shape, params=params)`